### DISCUSSION TITLE
Downloading Pull requests and commits

### DISCUSSION THREAD

**Discussion by Ek2100** [community/community#56598] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Why is there no GitHub API to Download Pull requests and commits?

**Comment by airtower-luna** [comment]
PRs are available [using the API](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28), to retrieve commits you can just [use standard Git commands](https://git-scm.com/book/en/v2) (fetch, etc.).

And if you're looking to find the commits *in a PR*, see: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally
